4337/contracts/Safe4337Module.sol

```diff
@@ -63,7 +75,6 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
         );
 
         address entryPoint = _msgSender();
```

Comment:
ubernit: Can you move this down into the `if` block where it is used?

Reply:
Actually, it is used before the `if` block for `validationData`.

Reply:
🤦 oops, sorry.